.eslintrc.json

```diff
@@ -54,6 +60,8 @@
         "react/jsx-filename-extension": "error",
         "react/prefer-stateless-function": "warn",
         "react/sort-comp": "warn",
-        "react/jsx-max-props-per-line": [1, { "when": "multiline" }]
+        "react/jsx-max-props-per-line": [1, { "when": "multiline" }],
+        "import/namespace": "off",
+        "import/default": "off"
```

Comment:
If we are making use of shareable configs, might I suggest my original approach of moving all the tests to their own folder and adding a folder config of the linter. It would make the approach much more cleaner and, in my opinion, make the code base more neat(for lack of a better word). I experimented with using extending the linter config, leading me to write this messy piece of code.

```
gulp.task('lint-test', () => {
    return gulp.src(['src/**/*.test.js', 'src/**/*.test.js'])
        .pipe(eslint({
            configFile: '.test-eslintrc.json'
        }))
        .pipe(eslint.format())
})

gulp.task('lint', ['lint-test'], () => {
...
}
```